src/theme.c

```diff
@@ -682,6 +705,9 @@ entry(struct theme *theme, const char *key, const char *value)
 	if (match_glob(key, "border.width")) {
 		theme->border_width = atoi(value);
 	}
+	if (match_glob(key, "padding.width")) {
```

Comment:
Do we need bounds checking here and for `window.button.spacing`?

Reply:
Yes, would make sense. We've not been super strict with that in other places, but it's good practice.

Reply:
Lower bound is easy, `0`, but upper bound is tricky? I could just use some arbitrary number like 50, but not sure if this makes sense and things might still fall apart. So just checking for non-negative numbers here and actually also at other places in this function?

Reply:
I've added a commit in between which clamps negative values to zero for some of the configuration options.